root/area/edit_form.tt

```diff
@@ -5,21 +5,21 @@
 
   <div class="half-width">
     <fieldset>
-      <legend>[%- l('Area details') -%]</legend>
+      <legend>[%- 'Area details' -%]</legend>
       [%- form_row_name_with_guesscase(r) -%]
-      [%- form_row_text_long(r, 'comment', add_colon(l('Disambiguation'))) -%]
```

Comment:
Should we bother removing all the `add_colon` calls and just add `:` into the strings?

Reply:
Commented with applicable suggestion: https://github.com/metabrainz/musicbrainz-server/pull/3120/files#r1424384912